galeracluster/source/sslsst.rst

```diff
@@ -0,0 +1,150 @@
+==================================
+SSL for State Snapshot Transfers
+==================================
+.. _`sslsst`:
+
+When you finish generating the SSL certificates for your cluster, you can begin configuring the node for their use.  Where :doc:`sslconfig` covers how to enable SSL for replication traffic and the database client, this page covers enabling it for :term:`State Snapshot Transfer` scripts.
+
+The particular method you use to secure State Snapshot Transfer scripts depends upon the particular method you choose to use on your cluster.
+
+.. note:: For Gelera Cluster, SSL configurations are not dynamic.  Since they must be set on every node in the cluster, if you are enabling this feature with a running cluster you need to restart the entire cluster.
+
+
+----------------------------------
+Enabling SSL for ``mysqldump``
+----------------------------------
+.. _`ssl-mysqldump`:
+
+The procedure for securing ``mysqldump`` is fairly similar to that of securing the database server and client and the standard method of enabling :term:`State Snapshot Transfer` script on your node.  Given that it connects through the database client, you can use the same SSL certificates you created for replication traffic.  In order to enable this for the entire cluster, you need to repeat this process on every node in the cluster.  
+
+#. From the database client you need to create a user for the State Snapshot Transfer script.
+
+   .. code-block:: mysql
+
+      CREATE USER 'sst_user'$'%' IDENTIFIED BY PASSWORD 'sst_password';
```

Comment:
This will create the user on all nodes, so there is no need to run this command for every node.